arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1084,98 +1049,109 @@ msgstr "Verkaufsabweichung"
 
-#: arbeitszeit_flask/templates/macros/coop_summary.html:26
-msgid "Current coordinator"
-msgstr "Aktuelle*r Koordinator*in"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:35
-msgid "Transfer coordination"
-msgstr "Koordination übergeben"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:42
-#: arbeitszeit_flask/templates/macros/list_coordinations_of_cooperation.html:5
-msgid "History of coordinators"
-msgstr "Abfolge von Koordinator*innen"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:47
-msgid "Cooperation price"
-msgstr "Kooperationspreis"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:57
-msgid "Participating plans"
-msgstr "Teilnehmende Pläne"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:79
-msgid "End cooperation"
-msgstr "Kooperation beenden"
-
-#: arbeitszeit_flask/templates/macros/coop_summary.html:90
-msgid "Individual price"
-msgstr "Individualpreis"
+#: arbeitszeit_flask/templates/macros/draft_form.html:9
+msgid "Specify product name and description."
+msgstr "Gib Produktname und Produktbeschreibung an."
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:22
+#: arbeitszeit_flask/templates/macros/draft_form.html:19
 msgid "Product description"
 msgstr "Produktbeschreibung"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:34
-#: arbeitszeit_web/formatters/plan_details_formatter.py:68
-msgid "Planning timeframe (days)"
-msgstr "Planungszeitraum (Tage)"
+#: arbeitszeit_flask/templates/macros/draft_form.html:28
+msgid ""
+"Choose a planning timeframe in which you are planning to produce and deliver "
+"your product."
+msgstr ""
+"Wähle einen Planungszeitraum, in dem du planst, dein Produkt herzustellen und "
+"zu liefern."
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:46
-#: arbeitszeit_web/formatters/plan_details_formatter.py:73
+#: arbeitszeit_flask/templates/macros/draft_form.html:31
+msgid "Planning timeframe"
+msgstr "Planungszeitraum"
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:41
+msgid ""
+"This value is used for informational purposes only, i.e. you do not have to "
+"adhere to any rules here. For example, if you plan to produce beverages, you "
+"are free to choose if your smallest delivery unit is one can or one six-pack "
+"of cans."
+msgstr ""
+"Dieser Wert dient nur zu Informationszwecken, du musst dich hier an keine "
+"Regeln halten. Wenn du z.B. planst, Getränke herzustellen, könnte deine "
+"kleinste Abgabeeinheit sowohl eine Dose als auch ein Sixpack sein."
+
+#: arbeitszeit_flask/templates/macros/draft_form.html:43
+#: arbeitszeit_web/formatters/plan_details_formatter.py:70
 msgid "Smallest delivery unit"
 msgstr "Kleinste Abgabeeinheit"
 
-#: arbeitszeit_flask/templates/macros/draft_form.html:49
+#: arbeitszeit_flask/templates/macros/draft_form.html:46
+msgid "E.g. 1 package, 1 kilogram, 20 bottles, 1 language lesson, etc."
+msgstr "Z.B. 1 Packung, 1 Kilogramm, 20 Flaschen, 1 Sprachstunde, etc."
```

Comment:
"Sprachstunde" does not seem to exist. "1 Sprachunterricht", "1 Stunde Sprachunterricht", "1 Lektion Sprachunterricht"?

Reply:
done